code/modules/flufftext/Hallucination.dm

```diff
@@ -1294,3 +1295,29 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	H.preparePixelProjectile(target, start)
 	H.fire()
 	qdel(src)
+
+/datum/hallucination/sleeping_carp
+
+/datum/hallucination/sleeping_carp/New(mob/living/carbon/C, forced = TRUE)
+	set waitfor = FALSE
+	..()
+	var/list/mobsyup = list()
+	for (var/mob/living/carbon/A in orange(C,2))
+		if (get_dist(C,A) < 2 && C != A)
+			mobsyup += list(A)
+	if (mobsyup.len == 0)
```

Comment:
```suggestion
	if (!LAZYLEN(mobsyup))
```
Even though the macro is just a placeholder for future possibly more reliable alternatives to length(A).